website/content/usage/_index.md

```diff
@@ -149,6 +149,28 @@ announcements. See
 [issue 1](https://github.com/metallb/metallb/issues/1) for more
 information.
 
+## eBPF and source IP preservation
+
+eBPF allows the CNI to run on the eBPF kernel layer. For our usecase, eBPF
+is an optimal solution to the IP Preservation problem highlighted in the aforementioned
+sections. It allows a CNI to implement a networking layer without having to
+rewrite the source IP of packets, enabling us to preserve the source IP, making
+network communication safer and additionally improving networking efficiency
+close to natively compiled in-kernel code. Please visit the
+[eBPF website](https://ebpf.io/) to read up more on the inner
+workings of eBPF and what other benefits it brings.
+
+### Calio
+
+Calico supports the eBPF dataplane natively. You can read up more about
+Calicos eBPF implementation and details [here](https://docs.projectcalico.org/about/about-ebpf).
+To enable the eBPF dataplane in Calico, follow [this guide](https://docs.projectcalico.org/maintenance/ebpf/enabling-bpf)
+
+### Cilium
+
+Another CNI that supports the eBPF dataplane is Cilium. You can read up more
+about its implementation and usage in the [Cilium docs](https://docs.cilium.io/en/latest/bpf/)
```

Comment:
Hi, Cilium developer here. :wave: 

The link here is pointing to the eBPF and XDP reference guide, which is a useful reference for eBPF but does not describe the Cilium implementation and usage of eBPF. Is there any reason not to just link to https://cilium.io here?

Also, for what it's worth I wouldn't necessarily describe Cilium as "Another CNI that supports the eBPF dataplane" since Cilium doesn't have another mode; Cilium was built from the ground up to leverage eBPF as we described in the blog post here: https://cilium.io/blog/2018/04/24/cilium-security-for-age-of-microservices . Given that this documentation is for a third party project, I think it probably makes sense to describe these external projects using the taglines on the front pages of the external project's website, so for instance for Cilium "Cilium provides eBPF-based Networking, Observability and Security". This is probably the more neutral way to describe the projects in the way that the projects themselves present themselves. For readers that are interested in more details, they can then follow the links and perform additional research on the project.

If you do want a more specific link to documentation regarding eBPF and client source IP preservation, the appropriate link is https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/#client-source-ip-preservation.

Reply:
Great Ideas! Would you mind creating a suggestion as to how you would like it to be written? I'm not very familiar with Cilium, so It'll be great if somebody with more expertise could write something about the inner workings of it.

Reply:
Yeah sure :-) Looking again at the context and also @tgraf's feedback, maybe something like the following would suffice..?

```suggestion
[Cilium](https://cilium.io) is a CNI that provides eBPF-based Networking, Observability, and Security. As part of its kube-proxy replacement implementation, Cilium is able to preserve the source IP when performing load balancing decisions across the cluster through [Direct Server Return (DSR)](https://cilium.io/blog/2020/02/18/cilium-17#kube-proxy-replacement-with-direct-server-return). For more information and guides to deploy Cilium, see the [Cilium Getting Started Guides](https://docs.cilium.io/en/stable/gettingstarted/).
```